packages/connect-common/files/coins.json

```diff
@@ -2198,6 +2198,25 @@
         }
     ],
     "misc": [
+        {
+            "blockchain_link": {
+                "type": "solana",
+                "url": ["solana.trezor.io"]
+            },
+            "curve": "ed25519",
+            "decimals": 9,
+            "is_testnet": false,
+            "name": "Solana",
+            "shortcut": "SOL",
+            "slip44": 501,
+            "support": {
+                "T1B1": false,
+                "T2B1": "2.6.1",
+                "T2T1": "2.6.1",
```

Comment:
note: need to check this. 2.6.1 is probably not right but I am not sure if these values are used at the moment